redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/DatabaseV4Endpoint.java

```diff
@@ -1,113 +1,105 @@
 package com.sequenceiq.redbeams.api.endpoint.v4.database;
 
-//import javax.validation.Valid;
-//import javax.validation.constraints.NotNull;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
 import javax.ws.rs.Consumes;
-//import javax.ws.rs.DELETE;
-//import javax.ws.rs.DefaultValue;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.DefaultValue;
 import javax.ws.rs.GET;
-//import javax.ws.rs.POST;
-//import javax.ws.rs.PUT;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.Produces;
-//import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
-//
-//import com.sequenceiq.cloudbreak.api.endpoint.v4.common.EnvironmentNames;
-//import com.sequenceiq.cloudbreak.api.endpoint.v4.database.requests.DatabaseTestV4Request;
-//import com.sequenceiq.cloudbreak.api.endpoint.v4.database.requests.DatabaseV4Request;
-//import com.sequenceiq.cloudbreak.api.endpoint.v4.database.responses.DatabaseTestV4Response;
-//import com.sequenceiq.cloudbreak.api.endpoint.v4.database.responses.DatabaseV4Response;
-//import com.sequenceiq.cloudbreak.api.endpoint.v4.database.responses.DatabaseV4Responses;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.EnvironmentNames;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.requests.DatabaseTestV4Request;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.requests.DatabaseV4Request;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.responses.DatabaseTestV4Response;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.responses.DatabaseV4Response;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.responses.DatabaseV4Responses;
 import com.sequenceiq.cloudbreak.doc.ContentType;
 import com.sequenceiq.cloudbreak.doc.ControllerDescription;
-//import com.sequenceiq.cloudbreak.doc.Notes;
-//import com.sequenceiq.cloudbreak.doc.OperationDescriptions.DatabaseOpDescription;
-import com.sequenceiq.redbeams.api.endpoint.v4.database.responses.EchoResponse;
-//
+import com.sequenceiq.cloudbreak.doc.Notes;
+import com.sequenceiq.cloudbreak.doc.OperationDescriptions;
+
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
-//
-//import java.util.Set;
 
-@Path("/v4/{workspaceId}/databases")
+import java.util.Set;
+
+@Path("/v4/databases")
```

Comment:
This is fine with me. Right now, the database server API still has the workspace ID in there, but I'm thinking that I should just drop it out. However, since I am also hearing that they could come back, IMO the workspace ID should remain in the underlying base model / database entity / repository code, to make things easier. I plan on just setting that to zero all the time.